blocks/embed/lite-vimeo-embed/lite-vimeo-embed.js

```diff
@@ -32,40 +33,47 @@ class LiteVimeo extends (globalThis.HTMLElement ?? class {}) {
     if (devicePixelRatio >= 2) devicePixelRatio *= .75;
     width = Math.round(width * devicePixelRatio);
     height = Math.round(height * devicePixelRatio);
+    if(imageUrl){
+      this.style.backgroundImage = `url("${imageUrl}_${width}x${height}")`;
+    }
+
+    let playBtnEl = this.querySelector('.ltv-playbtn');
+      // A label for the button takes priority over a [playlabel] attribute on the custom-element
+      this.playLabel = (playBtnEl && playBtnEl.textContent.trim()) || this.getAttribute('playlabel') || 'Play video';
+
+      if (!playBtnEl) {
+        playBtnEl = createTag('button', { aria-label: `${this.playLabel}`, class: 'ltv-playbtn'  }, null);
```

Comment:
```suggestion
        playBtnEl = createTag('button', { 'aria-label': `${this.playLabel}`, class: 'ltv-playbtn'  }, null);
```
lines 46-48 can be removed

Reply:
Import createTag from utils.js

Reply:
added indentation

Reply:
Thanks. Nit picky. :) but I see one more formatting issue LN: 59-61 on `lite-video-embed.js`

Reply:
@ryanmparrish my bad...fixed all such issues.